bench-tps/src/bench.rs

```diff
@@ -544,12 +545,23 @@ fn get_nonce_blockhash<T: 'static + BenchTpsClient + Send + Sync + ?Sized>(
     client: Arc<T>,
     nonce_account_pubkey: Pubkey,
 ) -> Hash {
-    let nonce_account = client
-        .get_account(&nonce_account_pubkey)
-        .unwrap_or_else(|error| panic!("{:?}", error));
-    let nonce_data = nonce_utils::data_from_account(&nonce_account)
-        .unwrap_or_else(|error| panic!("{:?}", error));
-    nonce_data.blockhash()
+    loop {
+        match client
+            .get_account_with_commitment(&nonce_account_pubkey, CommitmentConfig::processed())
+        {
+            Ok(nonce_account) => match nonce_utils::data_from_account(&nonce_account) {
+                Ok(nonce_data) => return nonce_data.blockhash(),
+                Err(err) => {
+                    info!("Couldn't get data from durable nonce account: {:?}", err);
```

Comment:
Is it ever possible to get here?

Reply:
Actually, never happens. Do you think it is better to just unwrap then?

Reply:
Yeah that probably makes the most sense